nslsii/EPS_devices/valves_and_shutters.py

```diff
@@ -0,0 +1,339 @@
+        The value displayed when the device is read that signifys state 2
+    state1_str : str, optional
+        The string value to be passed to ``RE(mv(device, val))`` to move to
+        state 1
+    state2_str : str, optional
+        The string value to be passed to ``RE(mv(device, val))`` to move to
+        state 2
+    num_retries : int, optional
+        This number of attempts at changing the state prior to raising an
+        error, the default is 1.
+    retry_sleep_time : float, optional
+        This is the time in seconds to wait between retries on changing the
+        state, the default is 0.5 s.
+    **kwargs : dict, optional
+        The kwargs passed to the ``ophyd.Device`` ``__init__`` method
+    '''
+
+    def __init__(self, *, prefix, name,
+                 state1_val='Open', state2_val='Closed',
+                 state1_str='open', state2_str='close',
+                 num_retries=1, retry_sleep_time=0.5,
+                 stop_str=None, **kwargs):
+
+        kwargs = dict(state1_pv_uid='Opn', state2_pv_uid='Cls',
+                      state1_val=state1_val, state2_val=state2_val,
+                      state1_str=state1_str, state2_str=state2_str,
+                      num_retries=num_retries,
+                      retry_sleep_time=retry_sleep_time,
+                      stop_str=stop_str, **kwargs)
+        super().__init__(prefix, name, **kwargs)
+
+
+class PhotonShutter(EPSTwoStateDevice):
+    '''An ``ophyd.Device`` for EPS driven photon shutters at NSLS-II
+
+    This is for use with NSLS-II EPS based photon shutters that have two
+    distinct states 'Open' and 'Close' and are actuated into these two states
+    by the PV's ``'{prefix}Cmd:Opn-Cmd'`` and ``'{prefix}Cmd:Cls-Cmd'``. This
+    is different from the ``GateValve`` class as it has, as standard, a larger
+    number of retries (``num_retries``) as default.
+
+    The basic use case is to define the device using the line:
+
+    ``my_shutter = PhotonShutter(prefix='a_PV_prefix', name='my_shutter')``
+
+    which can then be actuated by:
+    ``RE(mv(my_shutter, 'open')`` or
+    ``RE(mv(my_shutter, 'close')``
+
+    Parameters
+    ----------
+    prefix : str, keyword only
+        The PV prefix for all components of the device
+    name : str, keyword only
+        The name of the device as will be reported via read()
+    state1_val : float, int or str, optional
+        The value displayed when the device is read that signifys state 1
+    state2_val : float, int or str, optional
+        The value displayed when the device is read that signifys state 2
+    state1_str : str, optional
+        The string value to be passed to ``RE(mv(device, val))`` to move to
+        state 1
+    state2_str : str, optional
+        The string value to be passed to ``RE(mv(device, val))`` to move to
+        state 2
+    num_retries : int, optional
+        This number of attempts at changing the state prior to raising an
+        error, the default is 5.
```

Comment:
Let's put 10 as it was previously done via https://github.com/NSLS-II/nslsii/pull/47.
```suggestion
        error, the default is 10.
```